connectivity/FEATURE_BLE/source/cordio/source/GattServerImpl.cpp

```diff
@@ -657,7 +666,7 @@ ble_error_t GattServer::read(
     uint8_t cccd_index;
     if (get_cccd_index_by_cccd_handle(att_handle, cccd_index)) {
         if (connection == DM_CONN_ID_NONE) {
-            tr_error("Cannot read attribute (unknown connection ID)");
+            tr_error("unknown connection ID");
```

Comment:
you need context, when you read the trace how will you know what "unknown connection ID means"?
```
tr_error("Cannot read CCCD without a valid connection ID");
```

Reply:
The context is clearer from the trace:
`[ERR ][read]: unknown connection ID`

Reply:
the trace will look like this:
`{ERR ][BLGS]: unknown connection ID`

Reply:
You're correct. Context should be added to the traces. Please review commit 6216ad31f4a25e7c744e6b82bdc7c56a678ac8db when you get the chance.